lib/kubernetes-deploy/kubernetes_resource/cloudsql.rb

```diff
@@ -0,0 +1,66 @@
+module KubernetesDeploy
+  class Cloudsql < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def initialize(name, namespace, file)
+      @name, @namespace, @file = name, namespace, file
+    end
+
+    def sync
+      _, st = run_kubectl("get", type, @name)
+      @found = st.success?
+      @status = true
```

Comment:
Hmmm... Ideally, it would use the status field you explained doesn't exist yet, but maybe we could fake "Unknown"/"Provisioning"/"Provisioned"/"Available" via `@found == false`, `@found == true`, `mysql_service_exists?` and `deploy_succeeded?`? Or something like that? Or minimally "Unknown"/"Provisioned"?